### DISCUSSION TITLE
Accessory Grouping

### DISCUSSION THREAD

**Discussion by ImaLoTekGuy** [homebridge-simplisafe3/homebridge-simplisafe3#307] [Q&A] [answered]
I'm not sure if this is a Plugin or a HomeBridge issue, so starting here. 

Homekit is once again "grouping" like accessories - all doors together, all motion sensors together, all fire/CO alarms together, etc.

There used to be an option to "Show as Separate Tiles" for each grouped accessory but according to Apple Tech Support on a telephone call this morning this capability has been removed due to "security concerns".

Not sure this can be "fixed" in the plugin, or even HomeBridge, but wanted to check here first - TIA.

**Comment by shamoon** [comment, marked as answer]
Hi @ImaLoTekGuy , not sure if youre talking about a future HomeKit release, this is definitely not the case currently. Also I dont think this applies to this plugin, devices are not all one accessory so this would never be an issue, except for example with a camera + its motion sensor.

**Reply by ImaLoTekGuy** [reply]
Hi shamoon, thanks for the reply. They say a picture is worth a thousand words, here are two:

<img width="374" alt="Screen Shot 2022-02-21 at 11 12 01" src="https://user-images.githubusercontent.com/100111490/155001893-8991f9b3-6e29-4cd5-89bd-8741e6651a8c.png">

<img width="500" alt="Screen Shot 2022-02-21 at 11 13 28" src="https://user-images.githubusercontent.com/100111490/155000949-2301d41c-0687-435d-9ab4-0d0450b59d9d.png">

This is with MacOS Version 12.2.1 (21D62), and the same with iOS and iPadOS 15.3.1 (19D52). Home is Version 6.0 (557.4.1). There once was a "Show as Separate Tiles" option with each accessory, but that's gone. Based on a search, it appears the automatic grouping has been an on-again off-again "feature" from Apple for some time. Appears it's now "on-again". It will also force group a Smoke Sensor with a Carbon Monoxide Sensor, for instance, unless they're placed in separate "Rooms". Since they also no longer show "icons" in the Room area but only at the top when active, I prefer to keep them all in a SimpliSafe "Room".

**Reply by shamoon** [reply]
Oh yea that’s not the same “group accessories” thing you’re talking about, that screenshot is a change Apple made to HK a while back where it shows sensors in the “status bar” above, but they’re not a grouped accessory.

but from the screenshot it looks like you just need to change which rooms they’re in, once you do it’ll show just one up top.

And no, the plug-in has nothing to do with this display. There are other apps besides “Home” to control HomeKit btw, perhaps they would better suit you.

**Reply by ImaLoTekGuy** [reply]
Aha. I must've been running an older (pre-update) instance before I recently installed updates. That explains nicely, thanks for the assistance. Have a great day and thanks for the help and plugin. I've looked at some of the other apps and just haven't found one I like. Perhaps Apple will get the "Show as Separate Tiles" function back where I can ungroup, for instance, the smoke sensors from the CO sensors in the same room, or the different doors. Can always cross my fingers anyway!
